src/components/button/Button.tsx

```diff
@@ -1,28 +1,12 @@
-import type { ButtonHTMLAttributes, ReactNode } from 'react';
+import { type ButtonHTMLAttributes, type ReactNode } from 'react';
 
 interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
```

Comment:
```suggestion
interface ButtonProps
  extends ButtonHTMLAttributes<HTMLButtonElement>,
    PropsWithChildren {
  variant: 'large' | 'small' | 'ghost';
}
```
так будет лучше и не придется самому прописывать children вообще хорошая практика использовать возможности самого реакт